### PR TITLE
Animal Tweaks: Second attempt

### PR DESCRIPTION
**Pull request by NanakoAC**
New PR to fix merge problems

Copied Description

Overhauls animal AI a bit, adds autofeeding to animals that can eat, coded to have a minimal performance impact. They'll now seek food in the nearby area and automatically eat it, or beg for it if held by someone.Similar to what i did with ian, the scanning frequency gradually slows down to save performance, and jumps back to full if food is detected

Fixes mousetraps being useless, and whole ton of bugs

Changes drone laws afer discussion with administrative team, to better align with their expectations of drone behaviour

Requires some testing with containedsprites before its ready to go, so DNM until that is in

changes: 
  - rscadd: "Added automatic feeding to most animals which can eat. They will eat food nearby, and beg for food held by others. Animals may take a while to notice food near them."
  - tweak: "Tweaked a few animal sizes, metabolisms and meat amounts. Animals are a little less needy for food. Except dogs."
  - tweak: "Disciplining a dog with a rolled up newspaper will make it stop stealing food for a little while."
  - bugfix: "Fixed Mousetraps not triggering when placed on vents."
  - tweak: "Buffed mousetraps."
  - bugfix: "Dead mice don't squeak."
  - tweak: "Adjusted feedback messages for animals climbing onto people, and animals being fed by people."
  - tweak: "Cats can now walk on/under tables."
  - bugfix: "Improved animal AI, and fixed issues of runtime moving around while dead."
  - bugfix: "Made Runtime more responsive when thrown at mice."
  - imageadd: "Mice can now be worn on your shoulder (ear slot). Special thanks to superballs for making sprites"
  - bugfix: "Vent names when ventcrawling will no longer constantly shuffle around."
  - rscdel: "Maintenance drone lawset has been altered based on administrative feedback."
  - rscadd: "Drink dispensers and fax machines on tables, can now be walked under by small animals."

### CODE REVIEW

**Comment by skull132** [comment]
More merge errors. I have no clue where these came from this time around :l

**Comment by NanakoAC** [comment]
Merge errors fixed. This is the most complex of the unmerged PRs so do this one next

**Comment by skull132** [comment]
Apparently not?

File: code/modules/mob/mob_helpers.dm [hunk 1]
```diff
@@ -1127,4 +1127,50 @@ var/list/wierd_mobs_inclusive = list( /mob/living/simple_animal/construct,
 
 	else return null
 
+	
+#define POSESSIVE_PRONOUN	0
+#define POSESSIVE_ADJECTIVE	1
+#define REFLEXIVE			2
+#define SUBJECTIVE_PERSONAL	3
+#define OBJECTIVE_PERSONAL	4
+/mob/proc/get_pronoun(var/type)
```
Comment (skull132) [inline code/modules/mob/mob_helpers.dm]: As noted, this proc is rendered useless due to DM's text macros.

Comment (skull132) [inline code/modules/mob/mob_helpers.dm]: It should also utilize a switch() statement instead of if/elif x n/else. Because efficiency.

**Pull request merged**